stdlib/mexpr/ast.mc

```diff
@@ -274,12 +273,12 @@ end
 
 lang RecordPat
   syn Pat =
-  | PRecord {bindings : AssocMap} -- AssocMap String Pat
+  | PRecord {bindings : AssocMapStringPat}
```

Comment:
As #156 is merged, could we perhaps change this to `AssocMap String Pat` straight away?

Reply:
Definitely, will fix when I have time.

Reply:
Now fixed